### PR TITLE
Create AttributesBehavior.php

### PR DESCRIPTION
**Pull request by bscheshirwork**
Propose new behavior like "AttributeBehavior", but can accept multiply values

I think for multiply attributes 
we need use 
invert ([attribute => event]) syntax:
```php
[
    'class' => AttributesBehavior::className(),
    'attributes' => [
        'attribute1' => [
            ActiveRecord::EVENT_BEFORE_INSERT => new Expression('NOW()'),
            ActiveRecord::EVENT_BEFORE_UPDATE => \Yii::$app->formatter->asDatetime('0000-00-00'),
        ],
        'attribute2' => [
            ActiveRecord::EVENT_BEFORE_VALIDATE => [$this, 'storeAttributes'],
            ActiveRecord::EVENT_AFTER_VALIDATE => [$this, 'restoreAttributes'],
        ],
        'attribute3' => [
            ActiveRecord::EVENT_BEFORE_VALIDATE => $fn2 = [$this, 'getAttribute2'],
            ActiveRecord::EVENT_AFTER_VALIDATE => $fn2,
        ],
        'attribute4' => [
            ActiveRecord::EVENT_BEFORE_DELETE => function($event) {static::disabled() || $event->isValid = false;},
        ],
    ],
],
```

Let's discuss!

| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Links  | #14101, #9921

### CODE REVIEW

**Comment by samdark** [comment]
@yiisoft/core-developers thoughts?

**Comment by SilverFire** [comment]
Good idea, I'm voting for

**Comment by dynasource** [comment]
Im positive based on https://github.com/yiisoft/yii2/pull/14101#issuecomment-300434317

**Comment by blacksmoke26** [comment]
+1 for this

**Review by SilverFire — APPROVED** [review]

File: framework/behaviors/AttributesBehavior.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+namespace yii\behaviors;
+
+use Closure;
+use yii\base\Behavior;
+use yii\base\Event;
+use yii\db\ActiveRecord;
+
+/**
+ * AttributeBehavior automatically assigns a specified values to one or multiple attributes of an ActiveRecord
```
Comment (samdark) [inline framework/behaviors/AttributesBehavior.php]: `AttributeBehavior` → `AttributesBehavior`.

Comment (samdark) [inline framework/behaviors/AttributesBehavior.php]: a specified values → values specified

**Review by samdark — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**